assets/components/doubleHeading/doubleHeading.jsx

```diff
@@ -1,11 +1,21 @@
+// @flow
+
 // ----- Imports ----- //
 
 import React from 'react';
 
 
+// ---- Types ----- //
+
+type PropTypes = {
+  heading: string,
+  subheading: ?string,
```

Comment:
Following discussion offline, @svillafe reckons creating a component for a single tag is overkill. We're going to explicitly make `doubleHeading` for dual headings only, and kill off `simpleHeading` (and probably `infoText`), in another PR.